test/unit/gateways/global_collect_test.rb

```diff
@@ -362,6 +394,10 @@ def successful_authorize_response
     %({\n   \"creationOutput\" : {\n      \"additionalReference\" : \"00000014280000000092\",\n      \"externalReference\" : \"000000142800000000920000100001\"\n   },\n   \"payment\" : {\n      \"id\" : \"000000142800000000920000100001\",\n      \"paymentOutput\" : {\n         \"amountOfMoney\" : {\n            \"amount\" : 100,\n            \"currencyCode\" : \"USD\"\n         },\n         \"references\" : {\n            \"paymentReference\" : \"0\"\n         },\n         \"paymentMethod\" : \"card\",\n         \"cardPaymentMethodSpecificOutput\" : {\n            \"paymentProductId\" : 1,\n            \"authorisationCode\" : \"OK1131\",\n            \"card\" : {\n               \"cardNumber\" : \"************7977\",\n               \"expiryDate\" : \"0917\"\n            },\n            \"fraudResults\" : {\n               \"fraudServiceResult\" : \"no-advice\",\n               \"avsResult\" : \"0\",\n               \"cvvResult\" : \"0\"\n            }\n         }\n      },\n      \"status\" : \"PENDING_APPROVAL\",\n      \"statusOutput\" : {\n         \"isCancellable\" : true,\n         \"statusCode\" : 600,\n         \"statusCodeChangeDateTime\" : \"20160316205952\",\n         \"isAuthorized\" : true\n      }\n   }\n})
   end
 
+  def successful_authorize_response_airline_fields
```

Comment:
These two new responses appear to be very similar to the existing successful authorize and capture responses. The only difference that I'm seeing is the fraud fields. 

Is it necessary to add these two new responses? Does it make sense to instead update the two existing responses to include the (new?) fraud fields?

Reply:
The main difference I saw was the inclusion of `statusCategory` but it looks to me like that's now included in standard purchases too, so I'll collapse them.